pkg/kubelet/config/http_test.go

```diff
@@ -137,6 +137,7 @@ func TestExtractFromHTTP(t *testing.T) {
 						UID:       "111",
 						Name:      "foo" + "-" + hostname,
 						Namespace: "foobar",
+						SelfLink:  "/api/v1beta1/boundPods/foo",
```

Comment:
Sorry-- it's testapi, I got it backwards: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/testapi/testapi.go#L72